.github/workflows/ci.yml

```diff
@@ -258,19 +258,75 @@ jobs:
     needs: [build, conditional]
     if: needs.conditional.outputs.ci-store == 'true'
     runs-on: ubuntu-latest
-    timeout-minutes: 75
+    timeout-minutes: 150
     strategy:
       matrix:
-        db: [postgres, mysql, oracle, mssql, mariadb]
+        db: [aurora-postgres, postgres, mysql, oracle, mssql, mariadb]
       fail-fast: false
     steps:
       - uses: actions/checkout@v4
 
+      - id: aurora-init
+        name: Initialize Aurora environment
+        if: ${{ matrix.db == 'aurora-postgres' }}
+        run: |
+          wget https://github.com/ekalinin/awsping/releases/download/0.5.2/awsping.linux.amd64.tgz
+          tar xzvf awsping.linux.amd64.tgz
+          chmod +x awsping
+          ./awsping -verbose 1
```

Comment:
I'm torn on this. Removing this logic will certainly make the action simpler and remove an external dependency, however it will mean that their will be larger latencies between the runner and EC2 for uploading/downloading files if they're located in different regions. Given that we're not transferring huge files, maybe simple is better?

Reply:
I'd then go for "simpler is better" then. We could revert once we get into trouble with pinning to a specific region.